codespell_lib/data/dictionary.txt

```diff
@@ -24532,7 +24542,10 @@ sertificated->certificated
 sertificates->certificates
 sertification->certification
 servce->service
+servced->serviced
```

Comment:
This should probably suggest `served` too? (with `servcing` also suggesting `serving`)